radio/util/tts_de.py

```diff
@@ -50,8 +50,8 @@
                 (u"Fehler", "error", NO_ALTERNATE),
                 (u"Trim zentriert", "midtrim", 495),
                 (u"Poti zentriert", "midpot", 496),
-                (u"Maximale Trimmung erreicht", "maxtrim", NO_ALTERNATE),
-                (u"Minimale Trimmung erreicht", "mintrim", NO_ALTERNATE),
```

Comment:
There need to be 2 files, mintrim and maxtrim. You can have them both say the same thing.